en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes_ki.adoc

```diff
@@ -2,6 +2,19 @@
 
 The following list displays known issues and limitations for MuleSoft Composer. The list is sorted by the date in which the known issue or limitation appeared.
 
+== August 29, 2023
+
+=== Composer Platform
+
+[%header%autowidth.spread]
+
+|===
+
+|Known Issue |Workaround |ID
+
+|Some Composer flows appear stuck since the access token fails to refresh, which prevents events from being triggered as intended. This error can show up in different ways, such as `Failed to send request to`, `Failed to get next element`, or `UNKNOWN ERROR` exceptions. | Refresh the affected Composer flows manually. | W-13628406
```

Comment:
```suggestion
|Some Composer flows appear stuck because the access token fails to refresh, which prevents events from being triggered as intended. This error can show up in different ways, such as `Failed to send request to`, `Failed to get next element`, or `UNKNOWN ERROR` exceptions. | Refresh the affected Composer flows manually. | W-13628406
```

Reply:
(see since vs because in SG)